btrdb/experimental/bytewax_connectors.py

```diff
@@ -0,0 +1,124 @@
+import queue
+import threading
+import weakref
+
+from bytewax.inputs import DynamicInput, StatelessSource
```

Comment:
Not important yet, but will need to update the `pyproject.toml` optional dependencies to include `bytewax`